### ISSUE TITLE
Backdrop APIs in 1.3 still require the window to have some content

### ISSUE THREAD

**Issue by Balkoth** [microsoft/microsoft-ui-xaml#8416]
Seems like for the new XAML Backdrop APIs to work you have to have both a xaml and a xaml.cs file?

Just doing it like this crashes the application with code 3221226107 (0xc000027b):

```cs
public partial class App : Application
{
  public App()
  {
    this.InitializeComponent();
  }

  protected override void OnLaunched(Microsoft.UI.Xaml.LaunchActivatedEventArgs args)
  {
    m_window = new Window();
    m_window.SystemBackdrop = new DesktopAcrylicBackdrop();
    m_window.Activate();
  }

  private Window m_window;
}
```

_Originally posted by @Balkoth in https://github.com/microsoft/WindowsAppSDK/discussions/3406#discussioncomment-4955796_

**Comment by bpulliam** [comment]
The fix for this issue is in 1.3.1 that is now available.

**Comment by bpulliam** [comment]
1.3.1 = 1.3.230502000 (built May 2, 2023)
